webgui/scripts/codecompass/view/infoTree.js

```diff
@@ -79,6 +79,11 @@ function (ObjectStoreModel, TitlePane, declare, Memory, Observable, mouse,
     },
 
     onClick : function (item, node, event) {
+      if (item.onClick) {
+        item.onClick(item, node, event);
+        return;
+      }
```

Comment:
To suppress the InfoTree's default `onClick` implementation (which is opening a source file and jumping to an ASTNode's location, as seen below in the current function's code) if the tree's node is given a "custom" `onClick` handler (such as in `plugins/cpp/webgui/js/cppInfoTree.js`, see `setHandlingOfImplicit`). Implicit methods have no "ASTNode" locations that can be resolved to opening of a file, so a custom logic is to be done in case those nodes are clicked.